graalwasm/graalwasm-embed-c-code-guide/README.md

```diff
@@ -0,0 +1,196 @@
+# Embed C in Java Using GraalWasm
+
+The example below demonstrates how to compile a C function to WebAssembly and run it embedded in a Java application.
+
+## 1. Getting Started
+
+To complete this guide, you need the following:
+- [GraalVM JDK](https://www.graalvm.org/downloads/)
+- [Emscripten compiler frontend](https://emscripten.org/docs/tools_reference/emcc.html)
+- [Maven](https://maven.apache.org/)
+
+## 2. Setting up the Maven Project
+
+To follow this guide, generate the application from the [Maven Quickstart Archetype](https://maven.apache.org/archetypes/maven-archetype-quickstart/):
+
+```shell
+mvn archetype:generate -DarchetypeGroupId=org.apache.maven.archetypes -DarchetypeArtifactId=maven-archetype-quickstart -DarchetypeVersion=1.5 -DgroupId=com.example -DartifactId=demo -DinteractiveMode=false
+cd demo
+```
+
+### 2.1. Adding the Polyglot API and GraalWasm Dependencies
+
+The GraalVM SDK Polyglot API can be easily added as a Maven dependency to your Java project.
```

Comment:
GraalVM SDK Polyglot API > GraalVM Polyglot API

Reply:
We use this shorter name version more often now. (https://www.graalvm.org/jdk23/reference-manual/wasm/guides/embed-c-in-java/ still has the old name)